CMakeLists.txt

```diff
@@ -97,6 +97,9 @@ OPTION(USE_FREETYPE "Enable FreeType support" ON)
 OPTION(USE_PNG "Enable LibPNG support" ON)
 OPTION(USE_VORBIS "Enable Vorbis support" ON)
 
+OPTION(SDL_DISABLE_CONTROLLER_API "Don't use the sdl controller apis." OFF)
```

Comment:
```suggestion
OPTION(SDL_DISABLE_CONTROLLER_API "Don't use the SDL controller APIs." OFF)
```

Reply:
Maybe also expand description so people known when they could use those settings? Like "Don't use the SDL controller APIs. Used for ..."

Also, maybe invert the setting to `USE_SDL_CONTROLLER_API` to make it look like `USE_PNG`, `USE_VORBIS` above?

Reply:
Yeah no worries. Let me fix that.